### PR TITLE
added codecov badge 4pm-courses-1

### LINKED ISSUE #1: Add Codecov Badge to the Main Branch
- [x] Add Codecov Badge to the Readme

### PR DESCRIPTION
**Pull request by DistractedPrimate**
In this Pull Request, I added the Codecov badge to the README. 

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# team04-w22-5pm-courses
+# s22-4pm-courses
 
-[![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
+[![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses/branch/master/graph/badge.svg?token=8mmcEy6jrV)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses)
```
Comment (pconrad) [inline README.md:3]: Sorry... this is on `master`.

Can you try again with `main`

**Review by pconrad — DISMISSED** [review]
`main` instead of `master`

File: README.md (hunk 1 above)
Comment (DistractedPrimate) [reply README.md]: Yes just fixed!

**Review by roberto-garcia-0 — COMMENTED** [review]
Checked that the badge showed up on the browser and saw the percentage coverage.

**Review by roberto-garcia-0 — APPROVED** [review]

**Review by andrewhlu — APPROVED** [review]
Looks good!

**Pull request merged**